src/main/java/jenkins/bouncycastle/api/PEMEncodable.java

```diff
@@ -190,7 +211,7 @@ public static PEMEncodable decode(@Nonnull String pem, @Nullable final char[] pa
             } else if (object instanceof SubjectPublicKeyInfo) {
                 return new PEMEncodable(kConv.getPublicKey((SubjectPublicKeyInfo) object));
             } else if (object instanceof X509CertificateHolder) {
-                JcaX509CertificateConverter cConv = new JcaX509CertificateConverter().setProvider("BC");
+                JcaX509CertificateConverter cConv = new JcaX509CertificateConverter().setProvider(BOUNCY_CASTLE_PROVIDER.getName());
```

Comment:
is `setProvider(Provider)` not available in both versions?

```suggestion
            JcaX509CertificateConverter cConv = new JcaX509CertificateConverter().setProvider(BOUNCY_CASTLE_PROVIDER);
```